activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -1,4 +1,5 @@
 require "active_support/core_ext/string/strip" # for strip_heredoc
+require "active_support/time"
```

Comment:
Can we require only what we need. I believe only `active_support/core_ext/time/calculations` should be sufficient.